package.json

```diff
@@ -72,7 +72,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-test-renderer": "^18.2.0",
-    "release-it": "^15.10.0",
+    "release-it": "^16.1.3",
     "ts-node": "^10.4.0",
     "typescript": "^5.0.3"
```

Comment:
**Suggestion:** Consider using fixed versions for your dependencies. This can help prevent unexpected breaking changes from automatically updated packages.
```suggestion
    "release-it": "16.1.3",
    "ts-node": "10.4.0",
    "typescript": "5.0.3"
```